troubleshoot/elasticsearch/increase-tier-capacity.md

```diff
@@ -21,35 +16,22 @@ If a warning is encountered with not enough nodes to allocate all shard replicas
 
 To accomplish this, complete the following steps:
 
-:::::::{tab-set}
+:::::::{applies-switch}
```

Comment:
if the applies to says `stack`, we need procedures for all 4 deployment types. likely this will be a split between ECE/ECH and self/ECK, but we would need to verify in more detail